mlflow/server/handlers.py

```diff
@@ -799,6 +818,52 @@ def _delete_model_version_tag():
     return _wrap_response(DeleteModelVersionTag.Response())
 
 
+@catch_mlflow_exception
+def _download_artifact(artifact_path):
+    basename = os.path.basename(artifact_path)
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        tmp_path = os.path.join(tmp_dir, basename)
+        artifact_repo = _get_artifact_repo_mlflow_artifacts()
+        artifact_repo._download_file(artifact_path, tmp_path)
+        return send_from_directory(tmp_dir, basename)
+
+
+@catch_mlflow_exception
+def _upload_artifact(artifact_path):
+    head, tail = os.path.split(artifact_path)
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        tmp_path = os.path.join(tmp_dir, tail)
+        with open(tmp_path, "wb") as f:
+            chunk_size = 8192
+            while True:
+                chunk = request.stream.read(chunk_size)
+                if len(chunk) == 0:
+                    break
+                f.write(chunk)
+
+        artifact_repo = _get_artifact_repo_mlflow_artifacts()
+        artifact_repo.log_artifact(tmp_path, artifact_path=head or None)
+
+    return _wrap_response(UploadArtifact.Response())
+
+
+@catch_mlflow_exception
+def _list_artifacts_mlflow_artifacts():
```

Comment:
Can we add short / simple docstrings for these methods, particularly `_list_artifacts_mlflow_artifacts()`, to explain what these handlers do?

Reply:
Added simple docstrings!